readme.md

```diff
@@ -215,6 +215,7 @@ Once built, load it in the browser.
 
 ## Related
 
+- [GitHub Custom Tab Size](https://github.com/lukechilds/github-custom-tab-size) - Set custom tab size for code view
```

Comment:
Should be added to the end of the list. Would also be nice to mention in parens that Refined GitHub already hard-codes it to 4.